app/helpers/application_helper.rb

```diff
@@ -192,11 +192,16 @@ def crawlable_meta_data(opts = nil)
     opts ||= {}
     opts[:url] ||= "#{Discourse.base_url_no_prefix}#{request.fullpath}"
 
-    if opts[:image].blank? && (SiteSetting.default_opengraph_image_url.present? || SiteSetting.twitter_summary_large_image_url.present?)
-      opts[:twitter_summary_large_image] = SiteSetting.twitter_summary_large_image_url if SiteSetting.twitter_summary_large_image_url.present?
-      opts[:image] = SiteSetting.default_opengraph_image_url.present? ? SiteSetting.default_opengraph_image_url : SiteSetting.twitter_summary_large_image_url
-    elsif opts[:image].blank? && SiteSetting.apple_touch_icon_url.present?
-      opts[:image] = SiteSetting.apple_touch_icon_url
+    twitter_summary_large_image_url =
+      SiteSetting.site_twitter_summary_large_image_url
```

Comment:
![screenshot from 2018-11-14 08-51-19](https://user-images.githubusercontent.com/4335742/48452530-78cf1180-e7ea-11e8-9d2c-106f24e4f83f.png)

I personally follow the guideline of 80 chars per line to avoid horizontal scrolling on most places.